.env.example

```diff
@@ -36,6 +36,10 @@ SELF_HOSTING_ENABLED=false
 # `localhost` (or unset) is used for local development and testing
 HOSTING_PLATFORM=localhost
 
+# WIP: Secret key used for stuff?
```

Comment:
Used by rails directly https://api.rubyonrails.org/v7.1.3.2/classes/Rails/Application.html#method-i-secret_key_base

Reply:
For some additional context, this was originally introduced as an _alternative to_ using the `credentials:edit` for "one-click" Render deploys.  My thinking was that generating this key and saving it in an env var would be _easier_ for self-hosters than going through the process of securing `config/master.key`.

This may or may not be the best way to handle this Docker self hosting flow.  Suggestions around this are welcome!

https://github.com/maybe-finance/maybe/blob/36e8b144868c9b68b23d61c30ac9a04e29f23318/render.yaml#L29-L34

Reply:
> My thinking was that generating this key and saving it in an env var would be easier for self-hosters

Yes IMO is absolutely is 👍🏼